ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/Parameters.java

```diff
@@ -42,7 +42,7 @@
             "Affixes", "PreviousTag1", "PreviousTag2", "PreviousTagPatternLevel1",
             "PreviousTagPatternLevel2", "AggregateContext", "AggregateGazetteerMatches",
             "PrevTagsForContext", "PredictionsLevel1", "GazetteersFeatures", "WordEmbeddings",
-            "BrownClusterPaths", "Linkability"};
```

Comment:
These features are only populated by cross-lingual wikifier. There won't be any exception if they are not present.